Makefile

```diff
@@ -1,85 +1,41 @@
-DESCRIPTION = CS50 Library for Java
-MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-OLD_NAME = lib50-java
-NAME = libcs50-java
-VERSION = 2.0.0
-
-BUILD_DIR = build
-DEB_DIR = $(BUILD_DIR)/deb
-DOCS_DIR = docs
-JAR_DIR = $(BUILD_DIR)/usr/java/packages/lib/ext
-PACMAN_DIR = $(BUILD_DIR)/pacman
-RPM_DIR = $(BUILD_DIR)/rpm
-
-.PHONY: all
-all: clean deb pacman rpm
-
-.PHONY: bash
-bash:
-	docker run -i --rm -t -v "$(PWD):/root" cs50/cli
+DESTDIR ?= /usr/local/share/java
+VERSION = 2.0.1
 
 .PHONY: build
 build: clean Makefile src/edu/harvard/CS50.java
-	mkdir -p "$(JAR_DIR)"
-	javac -d "$(BUILD_DIR)" src/edu/harvard/CS50.java
-	jar cvf "$(JAR_DIR)/cs50.jar" -C "$(BUILD_DIR)" .
-	find "$(BUILD_DIR)" -type d -exec chmod 0755 {} +
-	find "$(BUILD_DIR)" -type f -exec chmod 0644 {} +
+	mkdir -p build/usr/share/java
+	javac -d build src/edu/harvard/CS50.java
+	cp src/edu/harvard/CS50.java build/edu/harvard
+	jar cvf build/usr/share/java/cs50.jar -C build .
```

Comment:
Looks like Java's indeed removing the extension mechanism per http://openjdk.java.net/jeps/220#Removed:-The-extension-mechanism. 

> The extension mechanism was introduced in JDK 1.2, which was released in 1998, but in modern times we have seen little evidence of its use. This is not surprising, since most Java applications today place the libraries that they need directly on the class path rather than require that those libraries be installed as extensions of the run-time system.

Should we instead just set `CLASSPATH`? In a sense, that's similar to having to specify `-lcs50` or adding it to `LDLIBS` in order to use `libcs50`, when students install it on their own machines.